develop/app/Anser/Services/v2/PaymentService.php

```diff
@@ -219,4 +219,74 @@ function (
             );
         return $action;
     }
+
+    /**
+     * show user wallet
+     *
+     * @param integer $u_key
+     * @return ActionInterface
+     */
+    public function getWallet(int $u_key): ActionInterface
+    {
+        $action = $this->getAction("GET", "/api/v2/wallet")
+            ->addOption("headers", [
+                "X-User-Key" => $u_key
+            ])
+            ->doneHandler(
+                function (
+                    ResponseInterface $response,
+                    Action $action
+                ) {
+                    $resBody = $response->getBody()->getContents();
+                    $data    = json_decode($resBody, true);
+                    $action->setMeaningData($data["data"]);
+                }
+            )
+            ->failHandler(
+                function (
+                    ActionException $e
+                ) {
+                    log_message("critical", $e->getMessage());
+                    $e->getAction()->setMeaningData(["message" => $e->getMessage()]);
+                }
+            );
+        return $action;
+    }
+
+    /**
+     * Add user wallet balance
+     *
+     * @param integer $u_key
+     * @param integer|null $addAmount
+     * @return void
+     */
+    public function createWallet(int $u_key, int $addAmount = null)
```

Comment:
1. I don't think the `createWallet` is a good method name, please change it.
2. Isn't the `addAmount` parameter required, why it need the initial value?

Reply:
About point 1, I will prefer you use the `increase` and `reduce` these clear meaning words.

Reply:
The comment raised by this comment has been resolved and the method name 'createWallet' has been changed to 'increaseWalletBalance'